Test/TestCase/SearchManagerTest.php

```diff
@@ -158,12 +159,49 @@ public function testRemove()
         $this->assertNull($view);
     }
 
+    public function testRemoveAll()
```

Comment:
It would be better to have at least the test cases for deleting documents as a part of this change, since we should at least prove that deletion works for a timeseries index. We can work on the test cases for the other methods on a timeseries index later.